### PR TITLE
Install azure by default

### LINKED ISSUE #772: Install the azure plugin by default
Once #771 is complete, the installation scripts should install the azure plugin by default, just like we install the other mixins by default. That way people can try out a plugin immediately.

The files to change are located in:

https://github.com/deislabs/porter/tree/master/scripts/install

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
* Install the azure plugin by default
*  Fix missing newline when installing packages
    We lost the \n when printing the success message after installing a mixin or plugin. It wasn't obvious until I went to update the install script and saw the messages printing all on the same line.
*  Update install page
    * Include information about plugins
    * Do not list each mixin/plugin included in the install script, link to the index page instead.

# What issue does it fix
Closes #772

# Notes for the reviewer
I need to do a stable release of the azure plugin before this will pass, otherwise `porter plugins install azure` will fail because it doesn't have a `latest` stable release to install.

# Checklist
- [x] Unit Tests
- [x] Documentation
  - [ ] Documentation Not Impacted

### CODE REVIEW

File: docs/content/install.md [hunk 1]
```diff
@@ -86,24 +86,33 @@ iwr "https://cdn.porter.sh/$VERSION/install-windows.ps1" -UseBasicParsing | iex
...
 
 ```console
 $ porter mixin install terraform
-installed terraform mixin
-v0.3.0-beta.1 (0d24b85)
+installed terraform mixin v0.3.0-beta.1 (0d24b85)
 ```
 
-All of the Porter mixins are published to `https://cdn.porter.sh/mixins/atom.xml`.
+All of the Porter authored mixins are published to `https://cdn.porter.sh/mixins/atom.xml`.
```
Comment (vdice) [inline docs/content/install.md]: nit: perhaps hyphenate for `Porter-authored`

File: docs/content/install.md [hunk 2]
```diff
@@ -86,24 +86,33 @@ iwr "https://cdn.porter.sh/$VERSION/install-windows.ps1" -UseBasicParsing | iex
...
+
+# Plugins
+
+We are working on building out [plugins](/plugins) to extend Porter and the stable
+plugins are installed by default.
+
+You can update an existing plugin, or install a new plugin using the `porter plugin
+plugin` command:
```
Comment (vdice) [inline docs/content/install.md]: Extra `plugin` here

File: docs/content/install.md [hunk 3]
```diff
@@ -86,24 +86,33 @@ iwr "https://cdn.porter.sh/$VERSION/install-windows.ps1" -UseBasicParsing | iex
...
+
+```console
+$ porter plugin install azure --version canary
+installed azure plugin v0.1.1-10-g7071451 (7071451)
+```
+
+All of the Porter authored plugins are published to `https://cdn.porter.sh/plugins/atom.xml`.
```
Comment (vdice) [inline docs/content/install.md]: same hyphenation suggestion as above for `Porter-authored`

File: scripts/install/install-linux.sh [hunk 4]
```diff
@@ -14,14 +14,16 @@ chmod +x $PORTER_HOME/porter
...
-$PORTER_HOME/porter mixin install helm --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install azure --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install terraform --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install az --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install aws --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install gcloud --version $MIXIN_PERMALINK
+$PORTER_HOME/porter mixin install exec --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install kubernetes --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install helm --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install azure --version $PKG_PERMALINK
```
Comment (vdice) [inline scripts/install/install-linux.sh]: Should we change this to `arm`?  (Would apply to the other install scripts...)

Comment (carolynvs) [reply scripts/install/install-linux.sh]: Thanks! Yes, that should have happened a while ago but no time like the present!

**Pull request merged**